terraform/deployments/tfc-configuration/rds.tf

```diff
@@ -0,0 +1,32 @@
+module "rds-integration" {
+  source  = "alexbasista/workspacer/tfe"
+  version = "0.9.0"
+
+  organization      = var.organization
+  workspace_name    = "rds-integration"
+  workspace_desc    = "This module manages AWS resources for creating RDS databases."
+  workspace_tags    = ["integration", "rds", "eks", "aws"]
+  terraform_version = "1.7.0"
+  execution_mode    = "remote"
+  working_directory = "/terraform/deployments/rds/"
+  trigger_patterns  = ["/terraform/deployments/rds/**/*"]
+
+  project_name = "govuk-infrastructure"
+  vcs_repo = {
+    identifier     = "alphagov/govuk-infrastructure"
+    branch         = "samsimpson1/rds"
```

Comment:
temporary or accidental? :)

Reply:
oh sorry didn't see auto-merge was on

Reply:
Temporary until I get the TF working properly in its new home